nncf/torch/graph/transformations/commands.py

```diff
@@ -50,6 +61,35 @@ def __str__(self):
     def __hash__(self):
         return hash(str(self))
 
+    def get_state(self) -> Dict[str, object]:
+        """
+        Returns a JSON-compatible dictionary containing a state of the object
+        """
+        state = {self._TARGET_TYPE_STATE_ATTR: self.target_type.get_state(),
+                 self._INPUT_PORT_STATE_ATTR: self.input_port_id}
+        if self.target_type in self._OPERATION_TYPES:
+            state[self._MODULE_SCOPE_STATE_ATTR] = str(self.module_scope)
+        elif self.target_type in self._HOOK_TYPES:
+            state[self._IA_OP_EXEC_CONTEXT_STATE_ATTR] = str(self.ia_op_exec_context)
+        return state
+
+    @classmethod
+    def from_state(cls, state: Dict[str, Any]) -> 'PTTargetPoint':
+        """
```

Comment:
Here's the convention to make algorithm data save-able and load-able from the checkpoints.
Pair of methods should be implemented. Whether `get_state` and `from_state` or `get_state` and `load_state`
`get_state` should return a dictionary with pure Python objects, 
`from_state` creates the object from scratch by using output of `get_state`, 
`load_state` - initializes object from the state